tests/integration/let-test.js

```diff
@@ -50,5 +56,58 @@ describeComponent('let', 'Integration: let helper', {
       expect(this.$('.before').text()).to.eq('a b');
       expect(this.$('.after').text()).to.eq('A B ab');
     });
+
+    it.only('supports components that recompute', function() {
```

Comment:
just noting here that this `.only` needs to be removed before landing the test

Also, the title should be "helpers that recompute"